### ISSUE TITLE
After loading opacity on image is 0

### ISSUE THREAD

**Issue by Peter74** [vvvmax/unitegallery#55]
Hi,
I do not use plugin of UniteGallery, I need to have everything under my control :)
When Unitegallery is initialised, it leave sometimes (not every time 3 times from 10) opacity 0 on images.
...after hover they are showed and opacity is changed to 1.
I cannot imagine, what is a problem.
here are pictures, that u can see it: http://s23.postimg.org/vzb7twj97/opacity.jpg , http://s2.postimg.org/44526748p/opacity1.jpg

**Comment by vvvmax** [comment]
HI, please send me a link so I could look. I don't understand from your
words what is the issue.

**Comment by Peter74** [comment]
I found, that when it happend there is error:
fillItemsArrayunitegallery.min.js:8:22478Error: The item should not be image type
fillItemsArrayunitegallery.min.js:7:22477
runGalleryunitegallery.min.js:7:18447
rununitegallery.min.js:8:10576
unitegalleryunitegallery.min.js:9:18005
(anonymous function)nfa.js:107
ijquery.min.js:1:27070
fireWithjquery.min.js:1:27829
readyjquery.min.js:1:29632
Jjquery.min.js:1:29811

**Comment by vvvmax** [comment]
HI. could you please send me a link to your website so I could look?

**Comment by vvvmax** [comment]
sorry that I didn't read emails, they lost often. you can write here:
valiano.ticksy.com
so it won't get lost

**Comment by Peter74** [comment]
Should I send or it is OK? ...I hope you will understand.
...main point is, that it is not initialized fully - I do not know how to explain more precisely, but if it happened, you can see, that there are missing overlay, icon...

**Comment by vvvmax** [comment]
Please send url, I"ll take a look. with the bug of cause.

**Comment by Peter74** [comment]
[http://nfa.cookies.sk/zivot-nfa/pellentesque-maximus-eget-velit-non-pulvinar-quisque-commodo-lectus-eget-suscipit-venenatis/](url)

**Comment by Peter74** [comment]
...just it is happened not every time, if you see images, just refresh few times. I do not understand, it is not first time, that I use your excellent gallery, but this I see first time.

**Comment by vvvmax** [comment]
HI, I don't see it happends, I refresh couple of times. sorry. it's weird.
if you know what I need to do in order to see it please tell.

**Comment by Peter74** [comment]
I do not know, why. I make video, just to show you, that is real not joke :) [https://youtu.be/BUYbgT7bPqM](url) ...I did not test on windows, but other galleries which I build I test also on iMac. I test it on Safari, Chrome, Firefox and Opera - result was same.

**Comment by Peter74** [comment]
Hi Max,

it looks, that I got it.
I did initialisation of gallery on even ready() - and as I wrote I had feeling, that not everything was initialised or so...
...I take some time and search in jQuery... I change event to load() and it looks that it help (load event is fired after all elements are fully loaded - which ready start directly, when DOM is ready - so pictures in this case could be still in progress...I use pictures for retina and that is 2 times bigger pictures then usually).
...I learn something new.

Anyway Max really thank you for your time. UniteGallery is number one for me - I really do not like use plugins for this, because I have full control over it.
...one more detail - thanks for extending documentation it helps even more.

I will close this case - I just want to know, that you got it!
